### DISCUSSION TITLE
Extending the Autocomplete for other use cases

### DISCUSSION THREAD

**Discussion by lasseitorp** [formkit/formkit#1582] [Ideas]
The Autocomplete input works great as its label suggests - an autocomplete. However, we see other use cases. A previous discussion (https://github.com/formkit/formkit/issues/1423) called for it to accept arbitrary inputs, and in the case I looked at, it needed a submit button in the field that could be tab-navigated to for full accessibility support.

Could these be options for the Autocomplete, or should it be a new FormKit input by itself?

A good working example is Google's search field (https://www.google.com/).

**Comment by Rednas83** [comment]
There is also a search input
https://formkit.com/inputs/search

Unfortunately it doesn't supports autocomplete or the ability to search inside documents😢 It currently only adds a clear button to clear the search input.

I did a request for a more advanced one
https://github.com/orgs/formkit/discussions/1480
